server/cloudapps/app_install.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package cloudapps
+
+import (
+	"errors"
+
+	"github.com/mattermost/mattermost-plugin-cloudapps/server/utils/md"
+)
+
+type InInstallApp struct {
+	App App
+}
+
+type OutInstallApp struct {
+	md.MD
+}
+
+func (r *registry) InstallApp(in *InInstallApp) (*OutInstallApp, error) {
+	if in.App.AppID == "" {
+		return nil, errors.New("app ID must not be empty")
+	}
+	r.apps[in.App.AppID] = &in.App
```

Comment:
Won't we get race condition here?
Also don't we need UninstallApp method as well?

Reply:
yyy, a total placeholder for now - will mark as such